docker-ci/parent/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Use latest stable debian release
+FROM debian:latest
+
+# Create user
+ARG USERNAME=appuser
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+RUN groupadd --gid $USER_GID $USERNAME  && \
+    useradd --uid $USER_UID --gid $USER_GID -m $USERNAME && \
+    apt update && \
+    apt install sudo -y && \
+    echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME && \
+    chmod 0440 /etc/sudoers.d/$USERNAME
+USER $USERNAME
+
+# Install packages
+RUN sudo apt install curl build-essential gcc protobuf-compiler libssl-dev pkg-config git apt-transport-https wget gnupg2 unzip -y
+    
+# Install Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y
```

Comment:
Did we decide that the `-sSf` was unnecessary?

Reply:
Nice `bash -s`

Reply:
i think its fine to keep -sSf, it silences except for errors and fail fast with no output on server fails returning an error code